docs/EtcdSecureCluster.md

```diff
@@ -15,21 +15,23 @@ Calico supports insecure and TLS/certificate-enabled etcd clusters.
 To use TLS-enabled etcd, the following environment variables need to be set
 before running any `calicoctl` command:
 
-- **`ETCD_AUTHORITY`**: The `<ip_address>:<port_number>` pair representing the 
+* **`ETCD_AUTHORITY`**: The `<hostname>:<port_number>` pair representing the 
  access point to the cluster. **Default**: 127.0.0.1:2379
-- **`ETCD_SCHEME`**: The http or https protocol used by the etcd datastore. 
+  * NOTE: When running Etcd with TLS enabled, the address of the ETCD_AUTHORITY 
+    must be a hostname value, NOT an IP address, such as `etcd-host:2379`.
+* **`ETCD_SCHEME`**: The http or https protocol used by the etcd datastore. 
  **Default**: http
-- **`ETCD_CA_CERT_FILE`**: The full path to the CA certificate file for the 
+* **`ETCD_CA_CERT_FILE`**: The full path to the CA certificate file for the 
  Certificate Authority that signed the etcd server key/certificate pair.
-- **`ETCD_CERT_FILE`**: The full path to the client certificate file for 
+* **`ETCD_CERT_FILE`**: The full path to the client certificate file for 
  accessing the etcd cluster.
-- **`ETCD_KEY_FILE`**: The full path to the client key file for accessing the 
+* **`ETCD_KEY_FILE`**: The full path to the client key file for accessing the 
  etcd cluster.
 
 For example:
 
 ```
-export ETCD_AUTHORITY=127.0.0.1:2379
+export ETCD_AUTHORITY=hostname:2379
 export ETCD_SCHEME=https
 export ETCD_CA_CERT_FILE=/path/to/ca.crt
```

Comment:
Can you make sure you provide an example that uses the paths created my the make target? I want to be able to run `make create-certs` (or whatever) then just copy and paste the exact environment I need for running calicoctl talking to secure etcd.